src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -218,91 +254,102 @@ def probsolve_ivp(
      [0.97]
      [0.98]
      [0.99]]
+
     """
-    stprl = _create_steprule(atol, rtol, step, firststep, ivp)
-    prior = _string2prior(ivp, which_prior, **kwargs)
-    gfilt = _create_filter(ivp, prior, method, **kwargs)
-    with_smoothing = method[-2] == "s" or method[-1] == "s"
-    solver = GaussianIVPFilter(ivp, gfilt, with_smoothing=with_smoothing)
+
+    # Normalize string inputs
+    method = method.upper()
+    which_prior = which_prior.upper()
```

Comment:
I suppose this is controversial

Reply:
I don't really see where this could lead to errors honestly, so I think it should be fine